android-core/src/main/java/com/mparticle/internal/ConfigManager.java

```diff
@@ -406,6 +408,11 @@ private synchronized void updateCoreConfig(JSONObject responseJSON, boolean newC
             editor.putString(KEY_PUSH_MESSAGES, sPushKeys.toString());
         }
 
+        if (responseJSON.has(KEY_DIRECT_URL_ROUTING)) {
+           directUrlRouting = responseJSON.optBoolean(KEY_DIRECT_URL_ROUTING);
+            editor.putBoolean(KEY_DIRECT_URL_ROUTING, directUrlRouting);
+        }
+
```

Comment:
Small formatting nit

```suggestion
        if (responseJSON.has(KEY_DIRECT_URL_ROUTING)) {
           directUrlRouting = responseJSON.optBoolean(KEY_DIRECT_URL_ROUTING);
           editor.putBoolean(KEY_DIRECT_URL_ROUTING, directUrlRouting);
        }

```